examples/ASTraitCollection/Podfile

```diff
@@ -0,0 +1,5 @@
+source 'https://github.com/CocoaPods/Specs.git'
```

Comment:
OHHHHHH!!! Did some example projects get moved to an examples_extra dir? Is that why my example app disappeared?! HA! I thought it got deleted by accident on a weird merge or something. 

Before I do anything drastic, I want to confirm: I should remove the example from the `examples` folder and update the one in `examples_extra`?